### PR TITLE
First web component

### PR DESCRIPTION
**Pull request by AlejaMC15**
## Summary
- Add new component Card

## Screenshot
Screenshot at Oct 06 15-00-41

## Trello Card
https://trello.com/c/zRnGY6Wx/19-wbooks-primer-webcomponent

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+##System
+package-lock.json
+package.json
```
Comment (dsandoval11) [inline .gitignore]: the package.json file shouldn't be here

File: components/card/card.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { LitElement, html } from "lit";
+import styles from "./card.styles";
+
+class CardComponent extends LitElement {
+  static get styles() {
+    return [styles];
+  }
+
+  render() {
+    console.log(this.data);
```
Comment (dsandoval11) [inline components/card/card.js]: remove unnecessary log

File: components/card/card.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+                  class="img-book"
+                  src="${item.img}"
+                  alt="img-book"
+                />
+              </div>
+              <div class="text">
+                <h3 class="title-book">${item.book_title}</h3>
+                <span class="author">${item.author}</span>
+              </div>
+            </div>
```
Comment (dsandoval11) [inline components/card/card.js]: this should be a web-component with its properties

File: components/card/card.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+class CardComponent extends LitElement {
+  static get styles() {
+    return [styles];
+  }
+
+  render() {
+    console.log(this.data);
+    return html`
+      <div class="container-book-list">
+        ${this.data.map(
```
Comment (dsandoval11) [inline components/card/card.js]: defines `data` as a property

File: components/card/maquetado.code-workspace [hunk 5]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (dsandoval11) [inline components/card/maquetado.code-workspace]: you can delete this file

**Comment by AlejaMC15** [comment]
Corrections made

File: components/book-list/book-list.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+  static get styles() {
+    return [styles];
+  }
+
+  render() {
+    return html`
+      <div class="container-book-list">
+        ${this.data.map(
+          (item) => html`
+            <div class="card">
```
Comment (dsandoval11) [inline components/book-list/book-list.js]: `<div class="card">` can be part of `card-component` 😬

Comment (AlejaMC15) [reply components/book-list/book-list.js]: Corrections made

**Review by dsandoval11 — APPROVED** [review]

**Pull request merged**